{{cookiecutter.profile_name}}/lsf_status.py

```diff
@@ -179,7 +188,11 @@ def get_status(self) -> str:
                 ),
                 file=sys.stderr,
             )
-            status = self._query_status_using_log()
+            try:
+                status = self._query_status_using_log()
+            except UnknownStatusLine as error:
+                print("UnknownStatusLine: {}".format(error), file=sys.stderr)
+                status = self.FAILED
```

Comment:
We have now to put calls to `_query_status_using_log()` in `try except` blocks because it may raise `UnknownStatusLine`. But then we catch `UnknownStatusLine` and we return `FAILED`. I think it would keep the code slightly cleaner if we don't raise `UnknownStatusLine`, but directly returned `FAILED` in `_query_status_using_log()` if this happens. As such, `_query_status_using_log()` does not throw exceptions, just return `status`, and we don't need to put its calls in `try except` blocks. But this is very minor...

Reply:
Yeah, I totally agree with what you're saying. My reason for doing it this way was more conceptual. It isn't `_query_status_using_log`'s job to decide what an error means in terms of a job status. It's job is to just return the status in the log.

Reply:
Yeah, you are right! I think we should keep as it is then!